feat(Navigation): Add Documentation Section Navegation (#19)
dd3tech/dd360-components-docs: docs/navigation/breadcrumbs.mdx

---
product: dd360-ds
title: React Breadcrumbs component
components: Breadcrumbs
---

<br />
<br />

# <span name="floating-nav">Breadcrumbs</span>

    The Breadcrumbs component displays the user's current location on the website using a breadcrumb path.

<br />

## <span name="floating-nav">Imports</span>

> <WindowEditor codeString="import { Breadcrumbs } from 'dd360-ds'" />

Reviewer: And don't forgot the import with the path.

```js
import Breadcrumbs from 'dd360-ds/Breadcrumbs'
```

## <span name="floating-nav">Usage</span>

<ContainerComponentDoc className="flex items-center">

>     <Breadcrumbs options={[{ name: 'First Path' }, { name: 'Second Path' }, { name: 'Third Path' }]} />

Reviewer: ```suggestion
    <Breadcrumbs options={[{ name: 'First location' }, { name: 'Second location' }, { name: 'Third location' }]} />
```

</ContainerComponentDoc>

<WindowEditor
    codeString="import { Button, ButtonGroup } from 'dd360-ds'

      <Breadcrumbs options={[{name: 'First Path'},{ name: 'Second Path'},{ name: 'Third Path'}]}/> "

/>

<br />

#### <span name="floating-nav">Options</span>

The prop option is an array of objects that contains information about the routes in the Breadcrumbs component, including the name, icon, and redirect function
of each route

<ContainerComponentDoc className="flex items-center">
    <Breadcrumbs
        options={[
            {

>                 name: 'Home',

Reviewer: Do you can add an example with the router.push o router.replace please?

                icon: () => <DynamicHeroIcon icon="HomeIcon" width="24" />,
                to: () => (window.location.href = '/')
            },
            {
                name: 'Docs',
                icon: () => <DynamicHeroIcon icon="BeakerIcon" width="24" />,
                to: () => (window.location.href = '/docs/get-started/getting-started')
            },
            {
                name: 'Breadcrumbs',
                icon: () => <DynamicHeroIcon icon="CalculatorIcon" width="24" />,
                to: () => (window.location.href = '/docs/navigation/breadcrumbs')
            }
        ]}
    />
</ContainerComponentDoc>

<WindowEditor

>     codeString={`import { Button, ButtonGroup } from 'dd360-ds'

Reviewer: Don't forgot import the `Breadcrumbs` component here

    import DynamicHeroIcon from 'dd360-ds/DynamicHeroIcon'

<Breadcrumbs
    options={[
        { name: 'Home', icon: () => <DynamicHeroIcon icon="HomeIcon" width="24" />, to: () => (window.location.href = '/') },
        {
            name: 'Docs',
            icon: () => <DynamicHeroIcon icon="BeakerIcon" width="24" />,
            to: () => (window.location.href = '/docs/get-started/getting-started')
        },
        {
            name: 'Breadcrumbs',
            icon: () => <DynamicHeroIcon icon="CalculatorIcon" width="24" />,
            to: () => (window.location.href = '/docs/navigation/breadcrumbs')
        }
    ]}
/>
`}/>

<br />

#### <span name="floating-nav">Separator</span>

> The separator prop allows you to change the default separator icon of the Breadcrumbs component

Reviewer: ```suggestion
The separator prop allows you to change the default separator icon between the options
```

<ContainerComponentDoc className="flex items-center">
    <Breadcrumbs options={[{ name: 'First Path' }, { name: 'Second Path' }, { name: 'Third Path' }]} separator="|" />
</ContainerComponentDoc>

<WindowEditor

>     codeString={`import { Button, ButtonGroup } from 'dd360-ds'

Reviewer: Don't forgot the import of the component `Breadcrumbs`

---
product: dd360-ds
title: React Breadcrumbs component
components: Breadcrumbs
---

<br />
<br />

# <span name="floating-nav">Breadcrumbs</span>

    The Breadcrumbs component displays the user's current location on the website using a breadcrumb path.

<br />

## <span name="floating-nav">Imports</span>

> The first alternative is recommended since they can reduce the application package.

Reviewer: ```suggestion
The first alternative is recommended since they can reduce the application bundle.
```
